test/suite/test_verbose03.py

```diff
@@ -0,0 +1,175 @@
+    # present.
+    expected_json_schema = {
+        'ts_sec': {'type': int, 'always_expected': True },
+        'ts_usec': {'type': int, 'always_expected': True },
+        'thread': {'type': str, 'always_expected': True },
+        'session_err_prefix': {'type': str, 'always_expected': False },
+        'session_dhandle_name': {'type': str, 'always_expected': False },
+        'session_name': {'type': str, 'always_expected': False },
+        'error_str': {'type': str, 'always_expected': False },
+        'error_code': {'type': int, 'always_expected': False },
+        'msg': {'type': str, 'always_expected': True },
+    }
+
+    # Validates the JSON schema of a given event handler message, ensuring the schema is consistent and expected.
+    def validate_json_schema(self, json_msg):
+        expected_schema = dict(self.expected_json_schema)
+        for field in json_msg:
+            # Assert the JSON field is valid and expected.
+            self.assertTrue(field in expected_schema, 'Unexpected field "%s" in JSON message: %s' % (field, str(json_msg)))
+            # Assert the type of the JSON field is expected.
+            self.assertEqual(type(json_msg[field]), expected_schema[field]['type'],
+                    'Unexpected type of field "%s" in JSON message, expected "%s" but got "%s": %s' % (field,
+                        str(expected_schema[field]['type']), str(type(json_msg[field])), str(json_msg)))
+            expected_schema.pop(field, None)
+        # Go through the remaining fields in the schema and ensure we've seen all the fields that are always expected be present
+        # in the JSON message
+        for field in expected_schema:
+            self.assertFalse(expected_schema[field]['always_expected'], 'Expected field "%s" in JSON message, but not found: %s' %
+                (field, str(json_msg)))
+
+    @contextmanager
+    def expect_event_handler_json(self, config, stdErr=False):
+        # Clean the stdout/stderr resource before yielding the context to the execution block. We only want to
+        # capture the verbose output of the using context (ignoring any previous output up to this point).
+        if stdErr:
+            self.cleanStderr()
+        else:
+            self.cleanStdout()
+        # Create a new connection with JSON format enabled.
+        if stdErr:
+            conn_config = 'json_output=[error]'
+        else:
+            conn_config = 'json_output=[message]'
+        if config != "":
+            conn_config += "," + config
+        conn = self.wiredtiger_open(self.home, conn_config)
+        # Yield the connection resource to the execution context, allowing it to perform any necessary
+        # operations on the connection (for generating the expected message output).
+        yield conn
+        # Read the contents of stdout/stderr to extract our messages.
+        output = self.readStderr(self.nlines) if stdErr else self.readStdout(self.nlines)
```

Comment:
Would it worth if we test that any output doesn't magically appear in stderr when we don't expect it?

Reply:
I moved the `validate_json_schema` into the base class c4992c2

On your second point:
> Would it worth if we test that any output doesn't magically appear in stderr when we don't expect it?
I don't believe we need to do this. If output occurs on stderr whilst we are expecting output in stdout (or vice versa), the python test suite runner will fail the test. It does this already for all our tests i.e. if there's contents in the stdout/stderr buffer the test will fail on unexpected [output](https://github.com/wiredtiger/wiredtiger/blob/develop/test/suite/wttest.py#L105) (e.g. `AssertionError: unexpected standard output`), unless the test explicitly flushes out the stdout/stderr buffers (e.g. `self.cleanStderr`, `self.cleanStderr`)